pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -168,7 +168,7 @@ func (plugin *kubenetNetworkPlugin) Init(host network.Host, hairpinMode kubeletc
 	if err != nil {
 		klog.Warningf("can't set sysctl %s: %v", sysctlBridgeCallIPTables, err)
 	}
-
+	// make use of cni.go -> getLoNetwork(binDirs []string) *cniNetwork  ???
```

Comment:
This seems like a leftover comment?

Reply:
Ups, right. Was a reminder for me what's next.

Reply:
@johscheuer can you remove this comment?

Reply:
Updated, could you take another look?